addons/hw_drivers/tools/helpers.py

```diff
@@ -462,23 +476,38 @@ def path_file(filename):
     elif platform_os == 'Windows':
         return Path().absolute().parent.joinpath('server/' + filename)
 
+
 def read_file_first_line(filename):
     path = path_file(filename)
     if path.exists():
         with path.open('r') as f:
             return f.readline().strip('\n')
 
+
 def unlink_file(filename):
     with writable():
         path = path_file(filename)
         if path.exists():
             path.unlink()
 
+
 def write_file(filename, text, mode='w'):
```

Comment:
maybe we could add a `filter_ext` parameter (same as `file_open`) with a default value of `.conf`?